parcels/field.py

```diff
@@ -1556,55 +1555,72 @@ def __init__(self, filename, dimensions, indices, netcdf_engine, timestamp=None,
         self.field_chunksize = field_chunksize
         self.chunk_mapping = None
         self.rechunk_callback_fields = rechunk_callback_fields
+        self.chunking_finalized = False
+        self.lock_file = lock_file
 
     def __enter__(self):
         if self.netcdf_engine == 'xarray':
             self.dataset = self.filename
             return self
+
         init_chunk_dict = self._get_initial_chunk_dictionary()
         try:
-            self.dataset = xr.open_dataset(str(self.filename), decode_cf=True, engine=self.netcdf_engine, chunks=init_chunk_dict)
+            # unfortunately we need to do if-else here, cause the lock-parameter is either False or a Lock-object (we we rather want to have auto-managed)
```

Comment:
Double 'we'  in the comment here?

Reply:
yes, grammar mistake - will be corrected. I left the comment in on purpose to make it understandable why `lock=False` is okay but `lock=True` is not, in that case.